src/tokens/TokenBalances.tsx

```diff
@@ -98,9 +111,29 @@ function TokenBalancesScreen({ navigation }: Props) {
     )
   }
 
+  const onPressNFTsBanner = () => {
+    ValoraAnalytics.track(HomeEvents.view_nft_home_assets)
+    navigation.navigate(Screens.WebViewScreen, {
+      uri: `https://nfts.valoraapp.com/?address=${walletAddress}&hide-header=true`,
```

Comment:
could we put the base url in networkConfig.ts? want to avoid hard coded config in hard to find places :)

Reply:
I edited this hard-coded part to use the network config. The commit is the below link!

https://github.com/valora-inc/wallet/pull/2632/commits/c9f0d1c339034c3be0e2af6c408ec89d7ab2ba68